x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -292,3 +293,11 @@ enrichWithClause
 lookupCommand
     : LOOKUP tableName=indexPattern ON matchFields=qualifiedNamePatterns
     ;
+
+matchCommand
+    : MATCH queryString
+    ;
+
+queryString
+    : string
```

Comment:
I think this should be. `QUOTED_STRING`, no ?

Reply:
They're equivalent:

```antlr
string
    : QUOTED_STRING
    ;
```

I thought that it would be better to use a parser definition rather than a lexer one - but I can change this if you think it's better 👍

Reply:
You're correct that they're currently equivalent, at the moment. On balance, I think it's better to use QUOTED_STRING, since it more clearly describes the intent.

Reply:
Sure! Addressed in 9f92087e2072a0417b0dca08b990c640b5b919b2